### PR TITLE
feat: remove deprecated pvc setup for atlantis data dir

### PR DESCRIPTION
**Pull request by jhaumont**
## what
- Removing deprecated pvc setup (related to this https://github.com/runatlantis/helm-charts/pull/173) 
- bump major version

## why
- Just to prepare it and don't forget it

### CODE REVIEW

**Comment by jhaumont** [comment]
Hello, I participate to Hacktoberfest, is it possible to add a `HACKTOBERFEST-ACCEPTED` label? 🙏

**Comment by jamengual** [comment]
sure, but we need to review it first

**Comment by jhaumont** [comment]
hello @jamengual 
Any update on this one? 🙏

File: charts/atlantis/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: v0.20.1
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 4.1.2
+version: 5.0.0
```
[the remark this answers is not in the document]
Comment (jhaumont) [reply charts/atlantis/Chart.yaml:5]: Hello @nitrocode I have remove 3 values deprecated as you requested, if you can take a look

**Comment by GenPage** [comment]
Hello @jhaumont please rebase your branch

**Comment by nitrocode** [comment]
Instead of removing the pvc, could we disable it instead in case people still want to use it? Is there a need to deprecate it?

**Comment by jhaumont** [comment]
> Instead of removing the pvc, could we disable it instead in case people still want to use it? Is there a need to deprecate it?

I don't understand. I don't remove the PVC here. I just remove the deprecated values to declare it. We have change the way to do it in another PR #173 with `volumeClaim:` values. And pvc is still activated by default (`enabled: true`).

**Comment by jhaumont** [comment]
@nitrocode any update regarding the merge of this one?

File: charts/atlantis/Chart.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 appVersion: v0.23.1
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 4.12.0
+version: 5.0.0
```
Comment (nitrocode) [inline charts/atlantis/Chart.yaml:6]: What do you folks think? @runatlantis/maintainers 

Are we ready to do a 5.x? Or should we wait for more interface refactoring before a major upgrade?

Comment (jamengual) [reply charts/atlantis/Chart.yaml:6]: I'm ok with that since this is a breaking change

Comment (jamengual) [reply charts/atlantis/Chart.yaml:6]: should we move forward with this?

Comment (GenPage) [reply charts/atlantis/Chart.yaml:6]: @jamengual We can, just need to coordinate version bumps with pending PRs

**Comment by GenPage** [comment]
Looks like the chart version is a conflict even though it bumps to 5.0

**Comment by jamengual** [comment]
@jhaumont could you fix the conflicts? Thanks.

**Comment by jhaumont** [comment]
> @jhaumont could you fix the conflicts? Thanks.

@jamengual done, you will merge it ? 🙏

**Pull request closed without merging**